packages/flame/lib/src/experimental/camera_component.dart

```diff
@@ -108,7 +111,10 @@ class CameraComponent extends Component {
 
   @override
   Iterable<ComponentPointPair> componentsAtPoint(Vector2 point) sync* {
-    final viewportPoint = point - viewport.position;
+    final viewportPoint = Vector2(
+      point.x - viewport.position.x + viewport.anchor.x * viewport.size.x,
+      point.y - viewport.position.y + viewport.anchor.y * viewport.size.y,
+    );
```

Comment:
is this the same as: `point - anchor.translate(viewport.position, viewport.size)` ?

Reply:
Could be, but I don't really understand what is `anchor.translate()`, and when it can be used.